### PR TITLE
Update Fastify and all other dependencies

### PR DESCRIPTION
**Pull request by gilles-crealp**
- Update to Fastify ^3.0.0
- Update tests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "fastify-no-icon",
-  "version": "3.0.0",
+  "version": "4.0.0",
```
Comment (jsumners) [inline package.json]: I will handle this. Please revert.

**Review by jsumners — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (gilles-crealp) [reply package.json]: No problem.

**Review by jsumners — APPROVED** [review]
Thank you. This looks good. I will merge and release when I am able.

**Comment by jsumners** [comment]
Sorry for the delay. This has been released as v4.0.0.

**Pull request merged**